test/e2e/cluster_upgrade.go

```diff
@@ -30,6 +30,7 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework"
 	"k8s.io/kubernetes/test/e2e/framework/ginkgowrapper"
 	"k8s.io/kubernetes/test/e2e/upgrades"
+	apps "k8s.io/kubernetes/test/e2e/upgrades/apps"
```

Comment:
@rmmh What i meant was i wish to have:
```
"k8s.io/kubernetes/test/e2e/upgrades/apps"
```
Not:
```
apps "k8s.io/kubernetes/test/e2e/upgrades/apps"
```

I synced with @foxish offline, and we concluded that this is not important. i will leave the comments here in case presubmit queue tests fail.